src/v/pandaproxy/client/test/CMakeLists.txt

```diff
@@ -14,9 +14,14 @@ rp_test(
   UNIT_TEST
   BINARY_NAME pandaproxy_client_fixture
   SOURCES
+    fetch.cc
```

Comment:
there would be some value in splitting the test out into a separate commit, but no big deal.